src/zac/notifications/subscriptions.py

```diff
@@ -16,35 +16,89 @@
 User = get_user_model()
 
 
+DESIRED = [
+    {
+        "path": reverse_lazy("notifications:callback"),
+        "channel": "zaken",
+        "filters": {},
+    },
+    {
+        "path": reverse_lazy("notifications:kownsl-callback"),
+        "channel": "kownsl",
+        "filters": {},
+    },
+]
+
+
 def subscribe_all(domain: str) -> List[Subscription]:
+    """
+    Subscribe to the configured channels, on all known NRCs.
+
+    Subscription is idempotent - the actual subscription will check if the desired
+    subscription already exists and only create it if it doesn't yet.
+
+    A system user/service account is created, whose token is used to authenticate
+    against the ZAC API.
+
+    :param domain: The domain (including protocol) where the application is hosted.
+      Required to build fully qualified callback URLs.
+    """
     user, _ = User.objects.get_or_create(username="notifications api")
     token, _ = Token.objects.get_or_create(user=user)
 
     subs = []
     for service in Service.objects.filter(api_type=APITypes.nrc):
-        subs.append(subscribe(service, token.key, domain))
+        subs += subscribe(service, token.key, domain)
     return subs
 
 
 def subscribe(service: Service, token: str, domain: str) -> Subscription:
```

Comment:
```suggestion
def subscribe(service: Service, token: str, domain: str) -> List[Subscription]:
```
Hier komt een list uit toch?

Reply:
good catch! signature is inderdaad veranderd.

Moet toch echt eens serieus gaan kijken naar MyPy :thinking: